IM_calculation/IM/im_calculation.py

```diff
@@ -100,10 +100,17 @@ def check_rotd(comps_to_store: Iterable[Components]) -> bool:
 def calculate_rotd(
     accelerations,
     comps_to_store: List[Components],
-    func=lambda x: np.max(np.abs(x), axis=1),
+    func=lambda x: np.max(np.abs(x), axis=-2),
```

Comment:
The default function should also be changed in intensity_measures.get_rotations

Reply:
Ahhh, helps if you add all the files you changed to your commit.

Reply:
lol